app/pages/admin/workflow-default-dialog.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Dialog from 'modal-form/dialog';
+
+const WorkflowDefaultDialog = ({ onSuccess, onCancel }) => (
+    <Dialog tag="div" closeButton={true} onCancel={onCancel}>
+      <div>
+        You are about to make the default workflow inactive,
+        which will remove the default workflow setting from this project.
+        The default workflow can be set in the edit workflows page of the project builder.
+      </div>
+      <br />
```

Comment:
I missed seeing the `<br />` last review. If the goal is to have a bit of space, then perhaps use a `<p>` around the text instead of a`<div>` and a `<br />`. Paragraphs are also block-level and browsers default to given them margins too.